setup.py

```diff
@@ -27,7 +27,7 @@
       url='https://github.com/LANL-Seismoacoustics/pisces',
       download_url='https://github.com/LANL-Seismoacoustics/pisces/tarball/0.3.2',
       keywords = ['seismology', 'geophysics', 'database'],
-      install_requires=['numpy','obspy>=1.0','sqlalchemy >1.0, <1.4','Click'],
+      install_requires=['numpy','obspy>=1.0','sqlalchemy<=1.4', <1.4','Click'],
```

Comment:
Oops, I guess also need to remove the <1.4 here.  Sorry!

Reply:
I saw that as soon as I pushed it